TestProject/output/GIFOutput.php

```diff
@@ -39,23 +39,22 @@ function outputBoard(Board $_board)
             }
         }
 
-        //Creates a new image which is going to be saved under -> TestProject/imageOutput/*.png
+        //Creates a new image in png-format
         imagepng($pngImage, "imageOutput/" . sprintf("image-%03d", $this->imageIndex) . ".png");
```

Comment:
Perhaps it would be good to use a unique gif-specific name here, otherwise the gif output will probably use already existing png images (from another run) and combine them to one gif.